Gemfile

```diff
@@ -1,11 +1,9 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
+gem 'byebug'
+gem 'capistrano'
+gem 'capistrano-one_time_key'
```

Comment:
Since we're only using Ruby for deployment and one-off scripts, I figured we could jettison the notion of bundler groups? Cc: @sul-dlss/infrastructure-team